src/test/java/com/selfxdsd/storage/SelfInvoicesITCase.java

```diff
@@ -116,4 +116,39 @@ public void cannotReturnActive() {
         ).invoices();
         invoices.active();
     }
+
+    /**
+     * SelfInvoices can return invoices of a Contract.Id.
+     */
+    @Test
+    public void returnsInvoicesOfContract() {
+        final Invoices invoices = new SelfJooq(new H2Database()).invoices();
+        final Contract.Id id = new Contract.Id(
+            "amihaiemil/docker-java-api",
+            "john",
+            "github",
+            "DEV"
+        );
+        final Iterable<Invoice> ofContract = () -> invoices.ofContract(id)
+            .iterator();
+        MatcherAssert.assertThat(ofContract, Matchers.iterableWithSize(1));
```

Comment:
@criske Can you add another Invoice to the SQL script so you have more than 1 invoices returned here? :D